volmdlr/wires.py

```diff
@@ -4373,17 +4373,17 @@ def from_step(cls, arguments, object_dict):
         return cls.from_center_normal(center, normal, radius,
                                       arguments[0][1:-1])
 
-    def to_step(self, current_id, surface_id=None):
+    def to_step(self, current_id, **kwargs):
```

Comment:
Can you specify in the docstrings what are the `kwargs`, please ?

Reply:
Done